src/kinova_gen3_base_config/config/control/picknik_kinova_gen3.ros2_control.yaml

```diff
@@ -54,6 +60,38 @@ joint_trajectory_controller:
       joint_7:
         goal: 0.05
 
+streaming_controller:
```

Comment:
I think this needs to be named servo_controller

Reply:
Okay, I didn't see it because I was looking at the diff. I think streaming_controller should be deleted